laterpay/application/Helper/String.php

```diff
@@ -204,4 +204,59 @@ public static function laterpay_json_encode( $data, $options = 0, $depth = 512 )
         return wp_json_encode( $data, $options, $depth );
     }
 
+    /**
+     * Get scrambled content.
+     *
+     * @param string $content Post Content.
+     *
+     * @return string
+     */
+    public static function get_scrambled_text( $content ) {
+        $final_content = '';
+        $tags          = [];
+
+        // Split the content by tags and store all words.
+        $words = preg_split('/<(.*?)>/', $content );
+
+        // Get All Tags from the content.
+        preg_match_all('/<(.*?)>/', $content, $matches, PREG_SET_ORDER, 0);
+
+        // Store all tags.
+        foreach ( $matches as $match ) {
+            if ( ! empty( $match[0] ) ) {
+                $tags[] = $match[0];
+            }
+        }
+
+        // Append the words and tags.
+        foreach ( $words as $key => $value ) {
+
+            $tag = ( ! empty( $tags[ $key ] ) ) ? $tags[ $key ] : '';
+
+            // Scramble the string before appending to final content.
+            $final_content  .= self::scramble_text( $value ) . $tag;
```

Comment:
Does it respect new lines as well? 

Shouldn't it be easy to use str_replace on original content after getting all words? @thrijith